plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -286,6 +375,13 @@ class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
             if (startingAnimator.canceled) {
               return
             }
+            // Prepare animator values
+            // Some animators might not have initial values and should be skipped from internal update
+            val animatorStartRequired = updateAnimatorValues(startingAnimator)
+            if (!animatorStartRequired) {
+              startingAnimator.skipped = true
```

Comment:
With this change the `lifecycleListeners` wont' be notified about `onAnimatorStarting` and neither of `onAnimatorEnding` (see [L436](https://github.com/mapbox/mapbox-maps-android/pull/1909/files#diff-4afad8e84850da48f89b60130a335fe9d19ce152334ab24201f1eacd792ee4f4R436)).

Are we okay with this behaviour change?

Reply:
I am getting a bit lost with the code so not sure what `internal update` means here... So maybe it's okay to skip the notifications I mentioned above... 😟❓

Reply:
Commentary is not updated from the intermediate PR state so it's confusing. I'll reword